x-pack/plugins/observability_solution/dataset_quality/public/components/dataset_quality/header.tsx

```diff
@@ -25,14 +32,18 @@ export default function Header() {
   return (
     <EuiPageHeader bottomBorder>
       <EuiPageHeaderSection>
-        <EuiTitle size="m">
-          <h2>{datasetQualityAppTitle}</h2>
-        </EuiTitle>
-        <EuiBetaBadge
-          label={betaBadgeLabel}
-          title={betaBadgeLabel}
-          tooltipContent={betaBadgeDescription}
-        />
+        <EuiFlexGroup gutterSize="s" alignItems="center">
+          <EuiText>
+            <h2>{datasetQualityAppTitle}</h2>
+          </EuiText>
```

Comment:
I wonder if it's a good style to have non `EuiFlexItem`s inside a flex group :shrug: